### DISCUSSION TITLE
Desktop light mode

### DISCUSSION THREAD

**Discussion by DinkleDorph** [ente-io/ente#2560] [Enhancements]
Would appreciate a desktop light mode! I am a light mode advocate :)

**Comment by agichim** [comment]
Same here, light mode advocate.

**Reply by forlaine** [reply]
Same here. Dark mode hurts my eyes to the point where desktop Ente is unusable for me. Thanks.

**Comment by urkos101** [comment]
same here.. light mode

**Comment by picajoso** [comment]
I'd be also really interested in the light mode for the desktop app, that would be great!

**Comment by beancaper** [comment]
I'd like a "Match System" option, please!
